saleor/graphql/checkout/types.py

```diff
@@ -855,6 +855,7 @@ def resolve_shipping_methods(root: models.Checkout, info: ResolveInfo):
         )
```

Comment:
shouldn't we add a test for that, that this decorator solved the actual problem? I believe that decorator is already tested.